PNA.adoc

```diff
@@ -168,38 +51,39 @@ The Portable NIC Architecture (PNA) is the P4 architecture that defines
 the structure and common capabilities for network interface controller
 (NIC) devices. PNA comprises two main components:
 
-1. A programmable pipeline that can be used to realize a variety of
+* A programmable pipeline that can be used to realize a variety of
 different "packet paths" going between the various ports on the device
 (e.g., network interfaces or the host system it is attached to), and
 
-2. A library of types (e.g., intrinsic and standard metadata) and
+* A library of types (e.g., intrinsic and standard metadata) and
 P4~16~ externs (e.g., counters, meters, and registers).
 
 PNA is designed to model the common features of a broad class of NIC
 devices. By providing standard APIs and coding guidelines, the hope is
 to enable developers to write programs that are portable across
-multiple NIC devices that are conformant to the PNA[^PNAPortability].
+multiple NIC devices that are conformant to the PNA[1].
```

Comment:
Insert whitespace before `[1]`.

Reply:
We might also want to use different text than `[1]`, since there are also auto-generated bibliography references that look like `[1]`, too.  Maybe `[Footnote 1]`, both where it is referenced from, and where the footnote is shown?